test/setupJest.ts

```diff
@@ -0,0 +1,39 @@
+import fetchMock from 'jest-fetch-mock';
+import makeServiceWorkerEnv from 'service-worker-mock';
+
+declare let global: any;
+const localeStack: string[] = [];
+fetchMock.enableMocks();
+
+const setupGlobal = () => {
+  Object.assign(global, makeServiceWorkerEnv());
+
+  global.CF_API_ENDPOINT = 'https://example.org';
+  global.CF_ACCT_ID = 'keymaster';
+  global.CF_LIST_ID = 'gozar';
+  global.CF_API_TOKEN = 'gatekeeper';
+
+  global.GSHEETS_API_ENDPOINT = 'https://example.org';
+  global.GSHEETS_ID = 'keymaster';
+  global.GSHEETS_API_KEY = 'gozarian';
+
+  global.DEFAULT_DEST_DOMAIN = 'https://any.org';
+
+  global.AUTH_TOKEN = 'zot';
+  pushLocale('en-US');
+};
+
+const pushLocale = (newLocale: string | null): string => {
```

Comment:
These are exposed to allow testing how things would work when there isn't a .toml setting.